src/ducks/sharing/share.styl

```diff
@@ -0,0 +1,60 @@
+@import 'cozy-ui'
+
+:global
+    label[for='pho-album-share-toggle']
+        cursor pointer
+
+:local
+    .pho-share-modal-content
```

Comment:
I kept class name from the cozy-photos-v3 version as it will be easier to factorize both codes soon.